homeassistant/components/smartthings/sensor.py

```diff
@@ -43,6 +43,13 @@
 }
 
 JOB_STATE_MAP = {
+    "airWash": "airwash",
```

Comment:
Should this be `air_wash`?
```suggestion
    "airWash": "air_wash",
```